scripts/machine_sizes.py

```diff
@@ -105,18 +107,35 @@ def get_host_stats(env_name):
     return host_stats_list, all_disks
 
 
-def get_host_usage_stats(env_name, days_past):
+def get_host_usage_stats(env_name, days_past, fixed_date):
     from datadog import api
-    import time
+ 
+    def datetime_timestamp(days_past, fixed_date):
+        if days_past:
+            end_time = int(time.time())
+            period = 24 * 3600 * days_past
+            start_time = end_time - period
+        if fixed_date:
+            st_range = ' 00:00'
+            et_range = ' 23:59'
+            
+            start_time = fixed_date.strftime('%Y-%m-%d') + st_range
+            start_time = time.strptime(start_time,'%Y-%m-%d  %H:%M') 
+            start_time= time.mktime(start_time)
```

Comment:
This can be replaced with just `start_time= time.mktime(fixed_date.timetuple())`